### PR TITLE
TWIS-33/forgot pw page

### PR DESCRIPTION
**Pull request by hlee0928**
# TWIS-33/forgot pw

## Link to Story

https://re-boot.atlassian.net/browse/TWIS-33

## Description

Created forgot password with email input & reset password button

## Screenshot(s) / GIF(s):

Screen Shot 2024-03-05 at 18 53 28

## Type of change

- [ ] Bug fix
- [x] New feature
- [ ] Dependency / Configuration update
- [ ] Documentation update

## Checklist:

- [x ] My code follows the project's style guidelines
- [ ] I have created a Storybook story for my component(s)
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { ForgotPW } from "./ForgotPW";
+import { Box } from "@mui/material";
+
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
```
Comment (Muradil-Erkin) [inline src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx:5]: all comments can be removed

File: src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+export default meta;
+type Story = StoryObj<typeof meta>;
+// More on writing stories with args: https://storybook.js.org/docs/writing-stories/args
+export const Default: Story = {
+  args: {
+    email: "",
+    password: "",
+    onSubmit: (email: string, password: string) => {
+      alert(`Email: ${email} and Password ${password}`);
+      alert(`Email: ${email} and Password: ${password}`);
```
Comment (Muradil-Erkin) [inline src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx:39]: you won't need 2 alerts

File: src/app/_components/RegistrationForm/ForgotPW/ForgotPW.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        </Typography>
+        <Typography fontWeight="medium">Your Email</Typography>
+        <TextField
+          id="email-input"
+          size="medium"
+          label="Enter your email"
+          variant="outlined"
+          sx={{pb:2}}
+        />
+        <Button variant="contained" sx={{p:2}}>Reset Password</Button>
```
Comment (Muradil-Erkin) [inline src/app/_components/RegistrationForm/ForgotPW/ForgotPW.tsx:22]: you should have a callback function that's passed in as props that this button onclick hooks up to

File: src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+export const Prefilled: Story = {
+  args: {
+    email: "totalrealemail123@gmail.com",
+    password: "Password1234",
+    onSubmit: (email: string, password: string) => {
+      alert(`Email: ${email} and Password ${password}`);
+      alert(`Email: ${email} and Password: ${password}`);
+    },
+  },
+};
```
Comment (Muradil-Erkin) [inline src/app/_components/RegistrationForm/ForgotPW/ForgotPW.stories.tsx:52]: your component don't currently have any props, so these `args` aren't doing anything. 
you also won't need 2 version of stories.

**Review by Muradil-Erkin — CHANGES_REQUESTED** [review]

**Review by Jack-Vu — COMMENTED** [review]
No additional change request then the current ones

**Review by lopezmarvin — APPROVED** [review]
LGTM!

**Comment by Tsai-Anthony** [comment]
.

**Review by Tsai-Anthony — DISMISSED** [review]
lgtm test

**Review by Jack-Vu — CHANGES_REQUESTED** [review]
Added some comments on your component your component story still needs some work with the actual story and arguments. Instead of just commenting them out have them reflect your component.

**Review by Muradil-Erkin — APPROVED** [review]

**Review by manzire — APPROVED** [review]
QA manzire test ,looks good !
image

**Review by laloclau — APPROVED** [review]
Checked email box and was able to click on the button.
image

**Pull request merged**